test/lib/tpunit++.hpp

```diff
@@ -274,6 +274,24 @@ namespace tpunit {
                                   m20, m21, m22, m23, m24, m25, m26, m27, m28, m29,
                                   m30, m31, m32, m33, m34, m35, m36, m37, m38, m39,
                                   m40, m41, m42, m43, m44, m45, m46, m47, m48, m49, name) { }
+
+         TestFixture(bool parallel, const char* name,
```

Comment:
👍 nice I like this, should make it easy to have both parallel and non-parallel tests without breaking everything 🙏